Download all .fastq files in a project/Download all .fastq files in a project.ipynb

```diff
@@ -0,0 +1,6347 @@
+{
```

Comment:
<p>This explanation could be a bit more clear - when you say "only returns so many results", how many is so many? Why does it only return partial results?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCellAtlas/data-consumer-vignettes/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>